src/views/header/ConservationView.js

```diff
@@ -14,23 +18,72 @@ const ConservationView = view.extend({
     // we need to wait until stats gives us the ok
     //@listenTo @model, "reset",@render
     this.listenTo(this.g.stats,"reset", this.render);
+    
+    var opts = _.extend( {}, { 
+      fillColor: '#660', // [ '#660', '#ff0'],
+      strokeColor: '#330',
+      maxHeight: 20,
+      rectStyler: function(rect, data) { return rect }
+    }, this.g.conservationConfig );
+        
+    this.fillColor = opts.fillColor;
+    this.strokeColor = opts.strokeColor;
+    this.maxHeight = opts.maxHeight;
+    this.rectStyler = opts.rectStyler;
+    
     return this.manageEvents();
   },
-
+  
+  // returns a function that will decide a colour 
+  // based on the conservation data it is given
+  colorer: function(colorRange) {
+    var colorer = function() { return "none" };
+        
+    if( typeof colorRange === 'string' ) {
+      colorer = function() { return colorRange };
+    }
+    else if( Array.isArray( colorRange ) ) {
+      if ( colorRange.length != 2 ) {
+        console.error( "ERROR: colorRange array should have exactly two elements", colorRange );
+      }
+      
+      if ( !d3_scale ) {
+        console.warn( "providing a [min/max] range as input requires d3 to be included - only using the first color" );
+        colorer = function(d) { return colorRange[0] };
+      }
+      else {
+        var scale = d3_scale.scaleLinear()
```

Comment:
shouldn't that be just "linear"?
https://github.com/mbostock/d3/wiki/Quantitative-Scales